protocol/avdl/keybase1/simple_fs.avdl

```diff
@@ -307,4 +307,17 @@ protocol SimpleFS {
    recorded by the server) of their most recent edit.
    */
   FSFolderEditHistory simpleFSFolderEditHistory(Path path);
+
+  // suppressNotifications suppresses KBFS notifications, particularly FSEvent
+  // and FSSyncEvent.
+  //
+  // Caller should call this again after nextSuppressIn to continue
+  // suppressing.  If no such call is made in 2*nextSuppressIn, KBFS will
```

Comment:
What are the units of `nextSuppressIn`?  Is it supposed to be a time duration (in which case you should have a `Sec` or `MSec` suffix)?

Reply:
It used to be the keybase1.DurationSec type. I changed it into int because I didn’t like it to be a floating number, but forgot to add the sec suffix to name. Will fix!